src/webview.cpp

```diff
@@ -131,6 +132,42 @@ QMenu* WebView::getHistoryForwardMenu() const
     return ret;
 }
 
+void WebView::downloadViewContent()
+{
+    try {
+        std::string url = this->url().path().toUtf8().constData();
+        if (url[0] == '/')
+            url = url.substr(1);
+
+        auto app = KiwixApp::instance();
+        auto archive = app->getLibrary()->getArchive(m_currentZimId);
+        auto entry = archive->getEntryByPath(url);
+        if (entry.isRedirect()) 
+            return;
+
+        auto item = entry.getItem(true);
+        auto mimeType = QByteArray::fromStdString(item.getMimetype());
+        mimeType = mimeType.split(';')[0];
+        
+        auto suggestedDir = app->getSettingsManager()->getPrevSaveDir();
+        auto suggestedPath = suggestedDir + "/" + this->url().fileName() + ".pdf";
+        if (mimeType == "text/html")
+        {
+            QString fileName = QFileDialog::getSaveFileName(
+                app->getMainWindow(), gt("save-file-as-window-title"),
+                suggestedPath, tr("PDF files (*.pdf)"));
```

Comment:
Do we really need a `gt` translation here? The filter seems like it should be a pretty standard translation.